scripts/comparator/run.sh

```diff
@@ -73,3 +73,6 @@ $comparator -input=$QUERY_FILE \
 -e=$END \
 -comparator=$COMPARATOR_WRITE \
 -regressionDir=$REGRESSION_DIR
+
+# Run PromQL testdata tests
+go test -v -timeout 30s -tags=compatibility -count=1 github.com/m3db/m3/src/query/test/
```

Comment:
I have added them together with comparator as it needs comparator running to do the tests. I think it would be best to run part of it as we won't need to run a separate infra for it. What do you think?